doc/ansible/dev/advanced_options.md

```diff
@@ -0,0 +1,36 @@
+## Advanced Options
+
+This document shows the advanced options available to a developer of a ansible operator.
+
+### Watches File Options
+
+The advanced features can be enabled by adding them to your watches file per GVK.
+They can go below the `Group`, `Version`, `Kind` and `playbook` or `role`.
```

Comment:
Is it relevant that they can be below them? I might instead note that they are at the same level as these parameters, unless order matters.

Reply:
Also the `group`, `version`, and `kind` parameters are not capitalized in the `watches.yaml` so I would capitalize them here either.

Reply:
The order does not matter, but I think from a best practices perspective, I would like this to be pretty standard that the g,v,k, playbook or role, then the extra values thoughts?